pkg/asset/installconfig/gcp/validation_test.go

```diff
@@ -138,11 +138,19 @@ func TestGCPInstallConfigValidation(t *testing.T) {
 			expectedError:  true,
 			expectedErrMsg: "network: Invalid value",
 		},
+		{
+			name:           "Invalid project ID",
+			edits:          editFunctions{invalidateProject},
+			expectedError:  true,
+			expectedErrMsg: "invalid project ID",
```

Comment:
try to include the `path` in the expectedErrMsg, makes it much clearer in terms of what we are testing.